GeneralStateTests/stMemoryTest/codeCopyOffset.json

```diff
@@ -2,10 +2,10 @@
     "codeCopyOffset" : {
         "_info" : {
             "comment" : "",
-            "filledwith" : "cpp-1.3.0+commit.dda985df.Linux.g++",
+            "filledwith" : "cpp-1.3.0+commit.97bd859a.Linux.g++",
             "lllcversion" : "Version: 0.4.14+commit.c2215d46.Linux.g++",
-            "source" : "src/GeneralStateTestsFiller/stCodeCopyTest/codeCopyOffsetFiller.json",
```

Comment:
you have to remove filler and filled test from that folder as well.
try running 
`./testeth -t GeneralStateTests`  
'./testeth -t BCGeneralStateTests'
and see that there are no errors